internal/terminal/handler/ghtrigger/ghtrigger.go

```diff
@@ -636,10 +660,10 @@ func (m *ModelGithubTrigger) optionSelector() string {
 	return windowStyle.Render(doc.String())
 }
 
-func (m *ModelGithubTrigger) sortTableItemsByName() {
+func (m *ModelGithubTrigger) sortTableItemsByID() {
 	rows := m.tableTrigger.Rows()
 	slices.SortFunc(rows, func(a, b table.Row) int {
-		return strings.Compare(a[2], b[2])
+		return strings.Compare(a[0], b[0])
 	})
 	m.tableTrigger.SetRows(rows)
 }
```

Comment:
It should keep same because the list can change every iteration.
For examle, first time:
```text
0 bool   abc
1 input  def
2 bool   hjk
```
Second run:
```text
0 input  def
1 bool   abc
2 bool   hjk
```

So, we need to sort by name. IDs just identifiers..